actions/retest/README.md

```diff
@@ -0,0 +1,15 @@
+# retest-action
+
+This is a github action built using the golang and the [github
+api](github.com/google/go-github). This main idea behind this one is to retest
```

Comment:
`This main idea...` has to be refactored.

Reply:
open for suggestion